### DISCUSSION TITLE
How can I set Feature Flags?

### DISCUSSION THREAD

**Discussion by patricksimonian** [istio/istio#53873] [Q&A]
Hi there Team,

I did read the accepted answer for a similar question posted in August but mine is a bit more generic. I was reading through the documentation and couldn't find any references to a list of env vars that could be set nor a best practice on applying them. My work is related to ensuring alpha/experimental features cannot be used and so I need to discover the different methods that they maybe applied to an installation. 

Thanks!

**Comment by howardjohn** [comment]
The majority of feature flags are environment variables on Istiod which can be enabled like `--set values.pilot.env.PILOT_ENABLE_IP_AUTOALLOCATE=true` during install
